test/peer_discovery_dns_SUITE.erl

```diff
@@ -85,18 +97,22 @@ end_per_testcase(_Testcase, Config) ->
 %% Test cases
 %% -------------------------------------------------------------------
 
+test_aaaa_record_hostname_discovery(_) ->
+    Result = rabbit_peer_discovery_dns:discover_hostnames(?DISCOVERY_ENDPOINT_RECORD_AAAA, true),
+    ?assert(string:str(lists:flatten(Result), "facebook.com") > 0).
```

Comment:
check a substring because they could change the sub-domain
```
rabbit_peer_discovery_dns:discover_hostnames("www.v6.facebook.com", true).
["v6-prod6-shv-09-frc3.facebook.com"]
```